models/kyberswap/binance_smart_chain/kyberswap_bsc_schema.yml

```diff
@@ -0,0 +1,96 @@
+version: 2
+
+models:
+  - name: kyberswap_bsc_trades
+    meta:
+      blockchain: bsc
+      sector: dex
+      project: kyberswap
+      contributors: duc.le@kyber.network
+    config:
+      tags: ['bsc', 'dex', 'trades', 'kyberswap']
+    description: >
+        Kyberswap trades on Binance Smart Chain across all contracts and versions. This table will load dex trades downstream.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
+      - &project
+        name: project
+        description: "Project name of the DEX"
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the DEX project"
```

Comment:
You should add `category` column as it is shown as `dex` or `aggregator`, `dex` will be understand as our pool provide liquidity (`dmm` or `elastic`), while `aggregator` means we collect liquidity from other source.